torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2745,6 +2744,19 @@ def sample_inputs_inner(self, device, dtype, requires_grad, **kwargs):
         ),
     )
 
+def sample_inputs_rbinops(op_info, device, dtype, requires_grad, supports_dtype_kwargs=True, **kwargs):
+    def _make_tensor_helper(shape, low=None, high=None):
+        return make_tensor(shape, device, dtype, low=low, high=high, requires_grad=requires_grad)
+
+    samples = [
+        SampleInput(_make_tensor_helper((S, S, S)), args=(3.14,)),
```

Comment:
Let's type the scalar argument like dtype, so 3, 3.14 if floating type, or 3.14 - 3.14j if complex